includes/class-wc-stripe-account.php

```
@@ -415,14 +415,63 @@ public function is_webhook_enabled() {
 		return $desired_events !== $existing_events;
 	}
 
+	/**
+	 * Checks if the API version of an existing webhook differs from our desired API version.
+	 *
+	 * @param object $existing_webhook The existing webhook object from Stripe.
+	 * @return bool True if API version differs, false if they match.
+	 */
+	private function does_webhooks_api_version_differ( $existing_webhook ) {
+		return self::get_webhooks_api_version() !== $existing_webhook->api_version; // @phpstan-ignore property.notFound
+	}
```

PHPMD (2.15.0)

[error] The method does_webhooks_api_version_differ is not named in camelCase. (undefined)

(CamelCaseMethodName)